.travis.yml

```
@@ -45,7 +37,6 @@ script:
       golint ./... | reviewdog -f=golint -reporter=github-pr-review
     fi
 after_success:
-  - cd ${TRAVIS_BUILD_DIR}/${VERSION_DIR}
   - |
```

if statement looks no longer needed.
```suggestion
  - bash <(curl -s https://codecov.io/bash)
```
and removing `REPORT_COVERAGE` would be better.